resources/lang/es/layout.php

```diff
@@ -54,14 +54,6 @@
             '_' => 'beatmapsets',
             'discussion' => 'modding',
         ],
-        'ranking' => [
```

Comment:
I think that's not easy for their translator to find that here are some missing sections, remove section block may easy to find